src/App.tsx

```diff
@@ -2,35 +2,45 @@ import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
 
-import Home from './components/Home';
-import ExamplePage from './components/ExamplePage';
-import NotFound from './components/NotFound';
-import NavBar from './components/NavBar';
-
-import { ThemeProvider } from '@material-ui/core';
+// Import antd stylesheets
+import 'antd/dist/antd.css';
```

Comment:
💭 Okay, so directly loading their distributed styles. Does this only have to be imported in App?